great_expectations/expectations/metrics/column_aggregate_metrics/column_value_counts.py

```diff
@@ -112,9 +113,10 @@ def _sqlalchemy(
         results = execution_engine.engine.execute(
             query.select_from(selectable)
         ).fetchall()
+        results_array = np.asarray(results)
         series = pd.Series(
-            [row[1] for row in results],
-            index=pd.Index(data=[row[0] for row in results], name="value"),
+            results_array[:, 1],
```

Comment:
Takes about half the time: https://gist.github.com/NathanFarmer/f2cd3ff62d3c1bf08858cdab642cb2a3